server-phoenix/lib/helios_web/schema/types/weather.ex

```diff
@@ -95,7 +101,7 @@ defmodule HeliosWeb.Schema.Types.Weather do
     field(:wind_gust, :float)
     field(:wind_deg, non_null(:integer))
     field(:clouds, non_null(:float))
-    field(:uvi, non_null(:integer))
+    field(:uvi, non_null(:float))
```

Comment:
Was this a bug? Does rails have the wrong type as well?

Reply:
Rails has it as an integer. I was getting an error on elixir because it was coming in as a float. I'll have to update the rails side as well.